Encounter Me/wwwroot/MapManager.js

```diff
@@ -0,0 +1,116 @@
+﻿var map;
+var markers = [];
+var geojsonLayer;
+
+function initialize(Lat, Lng)
+{
+    map = L.map('mapid').setView([Lat, Lng], 15);
```

Comment:
What does the "L" in "L.map" or "L.tileLayer" and in a few other places stands for? Does not  seem to be a class name.